modules/edge-identity/src/identity.rs

```diff
@@ -63,14 +55,50 @@ pub enum IdentityStage {
 
 #[cfg_attr(feature = "std", derive(Debug))]
 #[derive(Encode, Decode, PartialEq)]
-pub struct IdentityRecord<AccountId, BlockNumber> {
+pub struct IdentityRecord<AccountId, Moment> {
 	pub account: AccountId,
 	pub identity: Vec<u8>,
 	pub stage: IdentityStage,
-	pub expiration_time: Option<BlockNumber>,
+	pub expiration_time: Moment,
 	pub proof: Option<Attestation>,
-	pub verifications: Option<Vec<(AccountId, bool)>>,
-	pub metadata: Option<MetadataRecord>,
```

Comment:
I think we want to, in the long run, move metadata off-chain, but for now we want to keep it here. Note that since the storage elements are public, we do use the metadata, just in the front-end and not in other places on the chain.

Reply:
Ok, if you dont have database set up yet, i can understand the reason for this being on-chain.

Reply:
Agreed, since we're using this in our front-end, we do not want to remove this just yet. We will maintain a task to do so across our repos.